packages/connect-playwright/src/create-mock-router.ts

```diff
@@ -184,18 +202,26 @@ async function universalHandlerToRouteResponse({
 }) {
   const headers = await request.allHeaders();
   const abortSignal = new AbortController().signal;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- The Serializable type isn't exposed by Playwright
-  let body: any;
+
+  // Default body to an empty byte stream
+  let body: UniversalServerRequest["body"] = createAsyncIterable<Uint8Array>(
+    [],
+  );
```

Comment:
Nit: Not crucial here, but you can simply declare the variable without assigning anything, as long as all code paths assign before use.

Reply:
Yeah, the only issue was there was a code path that didn't assign it (if `buffer` was null), but I fixed that so that I didn't initialize it. And now the `AsyncIterable` is only created just-in-time if needed.